monkey/infection_monkey/post_breach/signed_script_proxy/windows/signed_script_proxy.py

```diff
@@ -1,21 +1,28 @@
 import os
 
 from infection_monkey.control import ControlClient
+from infection_monkey.utils.environment import is_windows_os
 
 TEMP_COMSPEC = os.path.join(os.getcwd(), "T1216_random_executable.exe")
 
 
 def get_windows_commands_to_proxy_execution_using_signed_script():
+    signed_script = ""
+
+    if is_windows_os():
```

Comment:
We shouldn't download files on command definition. We should define a command with "{filename}". Then override the run method to download the executable, and format the command. Either way, if you tested, let's keep it as is